src/threading/thread.cpp

```diff
@@ -311,7 +311,7 @@ bool Thread::bindToProcessor(unsigned int proc_number)
 
 bool Thread::setPriority(int prio)
 {
-#if USE_WIN_THREADS
+#ifdef _MSC_VER
```

Comment:
I added a special case for mingw.
Btw: That functions are currently not used, wouldn't it be better to completely remove them? If they are needed some day, they does still exist in git vcs.